src/test/java/org/minions/devfund/alejandro/tests/whackamole/WhackAMoleTest.java

```diff
@@ -0,0 +1,153 @@
+package org.minions.devfund.alejandro.tests.whackamole;
+import org.minions.devfund.alejandro.WhackAMole;
+import org.junit.Test;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * This class contain the test cases for WhackAMole class.
+ */
+
+public class WhackAMoleTest {
+
+    private final int numAttempts = 10;
+    private final int gridDimension = 4;
+    private int xPosition = 2;
+    private int yPosition = 2;
+    private final String expectedChar = "****\n****\n****\n****\n";
+    private final String expectedCharMoles = "****\n****\n**M*\n****\n";
+    private final String expectedCharWhacks = "****\n****\n**W*\n****\n";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)